### DISCUSSION TITLE
C runtime

### DISCUSSION THREAD

**Discussion by Uberme36** [maecry/asphyxia-cs2#25] [Q&A] [answered]
Hi! 
I was exploring source code of base and I can't find reason why we should use C runtime? What's the benefits of C runtime? Is it faster or what? Thanks for any relevant answer!

I'm talking about this file: [crt.h](https://github.com/maecry/asphyxia-cs2/blob/master/cstrike/utilities/crt.h)

**Comment by maecry** [comment, marked as answer]
https://www.unknowncheats.me/forum/cs-go-releases/585900-v2-qo0s-internal-cheat-base-template.html

you can read them here as this base is heavily inspired by qo0 base such things like crt also implemented for the same idea
